<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerCollectPointsModule</class>
 <widget class="qSlicerWidget" name="qSlicerCollectPointsModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module Template</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="ParameterLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ParameterNodeLabel">
       <property name="text">
        <string>Parameter Node:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ProbeTransformNodeLabel">
       <property name="text">
        <string>Transform Node:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="OutputNodeLabel">
       <property name="text">
        <string>Output Node:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="ProbeTransformNodeComboBox">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLLinearTransformNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="ParameterNodeComboBox">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLCollectPointsNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="OutputLayout">
       <item row="0" column="0">
        <widget class="qMRMLNodeComboBox" name="OutputNodeComboBox">
         <property name="nodeTypes">
          <stringlist>
           <string>vtkMRMLMarkupsFiducialNode</string>
           <string>vtkMRMLModelNode</string>
          </stringlist>
         </property>
         <property name="noneEnabled">
          <bool>true</bool>
         </property>
         <property name="renameEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="OutputVisibilityButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../qSlicerCollectPointsModule.qrc">
           <normaloff>:/Icons/PointVisible.png</normaloff>:/Icons/PointVisible.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ctkColorPickerButton" name="OutputColorButton">
         <property name="displayColorName">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="OutputDeleteButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../qSlicerCollectPointsModule.qrc">
           <normaloff>:/Icons/PointDelete.png</normaloff>:/Icons/PointDelete.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCheckablePushButton" name="CollectButton">
     <property name="text">
      <string>Collect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="AdvancedGroupBox">
     <property name="title">
      <string>Advanced</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QSpinBox" name="LabelCounterSpinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="MinimumDistanceLabel">
        <property name="text">
         <string>Minimum Distance:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="LabelCounterLabel">
        <property name="text">
         <string>Label Counter:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LabelBaseLineEdit">
        <property name="text">
         <string>P</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="LabelBaseLabel">
        <property name="text">
         <string>Label Base:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ctkSliderWidget" name="MinimumDistanceSlider">
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="VerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="ActionDeleteAll">
   <property name="icon">
    <iconset resource="../qSlicerCollectPointsModule.qrc">
     <normaloff>:/Icons/PointDeleteAll.png</normaloff>:/Icons/PointDeleteAll.png</iconset>
   </property>
   <property name="text">
    <string>Delete All Points</string>
   </property>
   <property name="toolTip">
    <string>Delete all points from the node</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCheckablePushButton</class>
   <extends>ctkPushButton</extends>
   <header>ctkCheckablePushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPushButton</class>
   <extends>QPushButton</extends>
   <header>ctkPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerCollectPointsModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerCollectPointsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ProbeTransformNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCollectPointsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>OutputNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCollectPointsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ParameterNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
